platform.md

```diff
@@ -1090,15 +1079,30 @@ For more information on build reproducibility see [https://reproducible-builds.o
 
 [run-image]
   reference = "<image reference>"
+  [target]
+  id = "<target identifer>"
+  os = "<OS name>"
+  arch = "<architecture>"
+  variant = "<architecture variant>"
+  [target.distribution]
+  name = "<OS distribution name>"
+  version = "<OS distribution version>"
 
 [build-image]
   reference = "<image reference>"
 ```
 
 Where:
-- `previous-image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
+- `image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
+- `image.metadata` MUST be the TOML representation of the layer [metadata label](#iobuildpackslifecyclemetadata-json)
 - `run-image.reference` MUST be either a digest reference to an image in an OCI registry or the ID of an image in a docker daemon
-- `previous-image.metadata` MUST be the TOML representation of the layer [metadata label](#iobuildpackslifecyclemetadata-json)
+- `run-image.target.id` is optional and MUST be the value of the label `io.buildpacks.id`
+- `run-image.target` contains the [target data](#target-data) for the image
+  - If target data is missing but the image contains the label `io.buildpacks.stack.id` with value `io.buildpacks.stacks.bionic`, the lifecycle SHALL assume the following values:
+    - `run-image.target.os = "linux"`
+    - `run-image.target.arch = "arm64"`
+    - `run-image.target.distribution.name = "ubuntu"`
+    - `run-image.target.distribution.version = "18.04"`
```

Comment:
```suggestion
  - If target distribution data is missing, it will be inferred from `/etc/os-release` for Linux images; furthermore, if the image contains the label `io.buildpacks.stack.id` with value `io.buildpacks.stacks.bionic`, the lifecycle SHALL assume the following values:
    - `run-image.target.os = "linux"`
    - `run-image.target.arch = "arm64"`
    - `run-image.target.distribution.name = "ubuntu"`
    - `run-image.target.distribution.version = "18.04"`
```